docs/security_conop/evaluation_artifacts/SCAP/Ssg_CentOS7_STIG.rst

```diff
@@ -0,0 +1,2516 @@
+-  Type: Finding
+-  Remediation:
+
+   This should be filed as a SIMP bug.
+
+   Note: Logging all ``chmod`` calls would likely result in a system denial
+   of service if done for all users.
+
+--------------
+
+-  Rule ID:   ``xccdf_org.ssgproject.content_rule_audit_rules_dac_modification_chown``
+-  Type: False Positive
+-  Notes:
+
+  -  Recommend SSG Feedback
+
+    -  The scan does not properly handle optimized rules which are
+       recommended by the prose guide
+
+-  Remediation:
+
+   .. code:: shell
+
+       grep chown /etc/audit/rules.d/*
+       /etc/audit/rules.d/50_base.rules:-a always,exit -F arch=b64 -S chown -S fchmod -S fchmodat -S fchown -S fchownat -S lchown -S setxattr -S lsetxattr -S fsetxattr -S removexattr -S lremovexattr -S fremovexattr -k perm_mod
+       /etc/audit/rules.d/50_base.rules:-a always,exit -F arch=b32 -S chown -S fchmod -S fchmodat -S fchown -S fchownat -S lchown -S setxattr -S lsetxattr -S fsetxattr -S removexattr -S lremovexattr -S fremovexattr -k perm_mod
+
+--------------
+
+-  Rule ID:   ``xccdf_org.ssgproject.content_rule_audit_rules_dac_modification_fchmod``
+-  Type: False Positive
+-  Notes:
+
+  -  Recommend SSG Feedback
+
+    -  The scan does not properly handle optimized rules which are
+       recommended by the prose guide
+
+-  Remediation:
+
+   .. code:: shell
+
+       grep fchmod /etc/audit/rules.d/*
+       /etc/audit/rules.d/50_base.rules:-a always,exit -F arch=b64 -S chown -S fchmod -S fchmodat -S fchown -S fchownat -S lchown -S setxattr -S lsetxattr -S fsetxattr -S removexattr -S lremovexattr -S fremovexattr -k perm_mod
+       /etc/audit/rules.d/50_base.rules:-a always,exit -F arch=b32 -S chown -S fchmod -S fchmodat -S fchown -S fchownat -S lchown -S setxattr -S lsetxattr -S fsetxattr -S removexattr -S lremovexattr -S fremovexattr -k perm_mod
+
+--------------
+
+-  Rule ID:   ``xccdf_org.ssgproject.content_rule_audit_rules_dac_modification_fchmodat``
+-  Type: False Positive
+-  Notes:
+
+  -  Recommend SSG Feedback
+
+    -  The scan does not properly handle optimized rules which are
+       recommended by the prose guide
+
+-  Remediation:
+
+   .. code:: shell
+
+       grep fchmodat /etc/audit/rules.d/*
+       /etc/audit/rules.d/50_base.rules:-a always,exit -F arch=b64 -S chown -S fchmod -S fchmodat -S fchown -S fchownat -S lchown -S setxattr -S lsetxattr -S fsetxattr -S removexattr -S lremovexattr -S fremovexattr -k perm_mod
+       /etc/audit/rules.d/50_base.rules:-a always,exit -F arch=b32 -S chown -S fchmod -S fchmodat -S fchown -S fchownat -S lchown -S setxattr -S lsetxattr -S fsetxattr -S removexattr -S lremovexattr -S fremovexattr -k perm_mod
+
+--------------
+
+-  Rule ID:   ``xccdf_org.ssgproject.content_rule_audit_rules_dac_modification_fchown``
+-  Type: False Positive
+-  Notes:
+
+  -  Recommend SSG Feedback
+
+    -  The scan does not properly handle optimized rules which are
```

Comment:
I think we have a ticket for a 'make the STIG shut up' version of the audit rules.